internal/ssdownload/download_manager.go

```diff
@@ -0,0 +1,243 @@
+// Copyright © 2022 Kaleido, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ssdownload
```

Comment:
S.S. Download 🚢

Reply:
This name looks funny whenever I read it, but I really have nothing better to offer. Could use simply `download`, but maybe that's too generic.

Reply:
Yeah, fought with this. Started with just `download`, but it's so coupled to shared storage that I added download (and used `sd` as its short name). Might go for `shareddownload`, which would put it next to `sharedstorage` in the components at least.